src/settings-compat-v7/_components.scss

```diff
@@ -68,6 +68,7 @@ $bdl-alert-basic-success: 4px solid $c-alert-basic-success !default;
 $bgc-alert-basic-success: $c-white !default;
 
 // Badge
+$c-badge-notification: $c-accent !default;
```

Comment:
We should let the same color as before: `$c-primary`. We'll set the `$c-accent` in the MT theme.